src/services/reactive-search.js

```diff
@@ -163,14 +163,18 @@ export const simpleQueryStringQuery = (value = "*") => {
   };
 };
 
-export const worksOnlyDefaultQuery = () => {
-  return {
-    query: {
-      match: {
-        "model.name": "Work",
-      },
+export const worksOnlyDefaultQuery = {
+  query: {
+    bool: {
+      must: [
```

Comment:
Does a must help with performance here? Just curious.

Reply:
@mathewjordan Not really, but if we wanted to add a second `must` condition, the setup is in place.  I was testing all kinds of stuff, but probably doesn't matter either way.